From bb925f636a8a3f662ef2a5f2826a7e0138dba266 Mon Sep 17 00:00:00 2001
From: Raphael Kubo da Costa <rakuco@FreeBSD.org>
Date: Wed, 31 Dec 2014 01:15:28 +0200
Subject: [PATCH] Rename INSTALL_PREFIX to KADU_INSTALL_PREFIX.

In CMake 3.1.0, INSTALL_PREFIX is now an optional argument to
configure_package_config_file(), so our call was confusing the
function's argument parsing code.

Fix it by renaming INSTALL_PREFIX to KADU_INSTALL_PREFIX.

Signed-off-by: Raphael Kubo da Costa <rakuco@FreeBSD.org>
---
 CMakeLists.txt            |    4 ++--
 cmake/KaduConfig.cmake.in |    2 +-
 2 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index ef624a9..3f23c71 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -195,8 +195,8 @@ foreach (install_var ${install_vars})
 	endif ()
 endforeach ()
 
-set (INSTALL_PREFIX "${CMAKE_INSTALL_PREFIX}")
-set (path_vars INSTALL_PREFIX)
+set (KADU_INSTALL_PREFIX "${CMAKE_INSTALL_PREFIX}")
+set (path_vars KADU_INSTALL_PREFIX)
 if (UNIX AND NOT APPLE)
 	list (APPEND path_vars DESKTOP_FILE_DIR)
 endif ()
diff --git a/cmake/KaduConfig.cmake.in b/cmake/KaduConfig.cmake.in
index 2e22aa4..6a3c683 100644
--- a/cmake/KaduConfig.cmake.in
+++ b/cmake/KaduConfig.cmake.in
@@ -20,7 +20,7 @@ endif ()
 set (KADU_INCLUDE_DIR "@PACKAGE_INCLUDE_DIR@")
 set (KADU_INCLUDE_DIRS "@PACKAGE_INCLUDE_DIR@" "@PACKAGE_INCLUDE_DIR@/kadu-core")
 set (KADU_SDK_DIR "@PACKAGE_SDK_DIR@")
-set (KADU_INSTALL_PREFIX "@PACKAGE_INSTALL_PREFIX@")
+set (KADU_INSTALL_PREFIX "@PACKAGE_KADU_INSTALL_PREFIX@")
 
 set (KADU_INSTALL_SDK "@INSTALL_SDK@")
 
-- 
1.7.1

